angular-workspace/projects/ni/nimble-angular/table-column/anchor/nimble-table-column-anchor-navigation-guard.directive.ts

```diff
@@ -33,11 +33,11 @@ export class NimbleTableColumnAnchorNavigationGuardDirective {
 
```

Comment:
Looks like the groupHeaderValue needs the same treatment? 
https://github.com/ni/nimble/blob/db6fc1fd69e6919797f500197c0647a24c5f9f61/packages/nimble-components/src/table-column/base/group-header-view/index.ts#L25